frontend/src/features/public-form/components/FormFields/SectionSidebar.tsx

```diff
@@ -71,24 +78,32 @@ export const SectionSidebar = (): JSX.Element => {
 
   return (
     <Box
+      as="nav"
+      aria-label="Form sections"
       flex={1}
       d={{ base: 'none', md: 'initial' }}
       minW={sectionScrollData.length > 0 ? '20%' : undefined}
     >
-      <VStack
+      <UnorderedList
         pos="sticky"
         top={sectionTopOffset}
         spacing="1.25rem"
         alignSelf="flex-start"
-        align="flex-start"
+        alignItems="flex-start"
+        marginInlineStart={0}
         marginEnd="1rem"
       >
         {sectionScrollData?.map((d) => (
-          <Flex key={d._id} align="center">
+          <ListItem key={d._id} listStyleType="none">
             <SidebarLink isActive={activeSectionId === d._id} sectionMeta={d} />
-          </Flex>
+          </ListItem>
         ))}
-      </VStack>
+      </UnorderedList>
+      {navigatedSectionTitle && (
+        <VisuallyHidden aria-live="assertive" aria-atomic>
+          Navigated to {navigatedSectionTitle} section
+        </VisuallyHidden>
```

Comment:
should we add this to the mobile drawer too?

Reply:
no need, since its actually hidden to screen readers, and because a drawer is used, when the drawer closes the element that triggered the drawer will be focused again... haha

Reply:
ohhh i see ok!